src/wallet/rpc/spend.cpp

```diff
@@ -983,6 +983,23 @@ static RPCHelpMan bumpfee_helper(std::string method_name)
                              "are replaceable).\n"},
                     {"estimate_mode", RPCArg::Type::STR, RPCArg::Default{"unset"}, "The fee estimate mode, must be one of (case insensitive):\n"
                              "\"" + FeeModes("\"\n\"") + "\""},
+                    {"extra_outputs", RPCArg::Type::ARR, RPCArg::Default{UniValue::VARR}, "Additional outputs (key-value pairs) in the same format\n"
+                             "as the \"outputs\" argument to \"send\": each address can only appear once and there can only be\n"
+                             "one \"data\" object.\n",
+                         {
```

Comment:
This is pretty much copied from `send`. Not sure how to extract it somewhere and reuse in both places.

Reply:
I think its okay because they are already used at multiple places: `send()`, `sendall()` and `walletcreatefundedpsbt()`

Reply:
You can de-duplicate this in the same way that `FundTxDoc` works. Have a look at that and its usage in various help texts for some guidance.

Reply:
@achow101 thanks! Done.